Src/library/memory_analyzer/widgets/ma_window.e

```diff
@@ -38,17 +39,14 @@ feature {NONE} -- Initialization
 
 			create_all_widgets
 			create_object_routes_panel_objects
+			create timer
```

Comment:
Looks like those changes are incorrect. You should simply redefine `create_interface_objects` and add creations there, no?